floodviz/map_utils_tests.py

```diff
@@ -0,0 +1,171 @@
+from floodviz.map_utils import site_dict, get_sites_geojson
```

Comment:
Should have said something early. It's our practice here to name put our tests in a directory and call the file test_*. So please move this to floodviz/tests/test_map_utils.py